pkg/serverless/metrics/metric.go

```diff
@@ -124,3 +129,9 @@ func buildBufferedAggregator(multipleEndpointConfig MultipleEndpointConfig, forw
 func buildMetricBlocklist(userProvidedList []string) []string {
 	return append(userProvidedList, invocationsMetric)
 }
+
+// Need to account for duplicate metrics when using the proxy.
+// Can be removed when the proxy is removed
```

Comment:
Nit: I would remove `// Can be removed when the proxy is removed`
We don't really know if and when this will happen, and if and when we do remove it, we should be able to figure it out from the name of the function.